### DISCUSSION TITLE
Full screen icon

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#739] [Request: Icon] [answered]
Hello, 
We are working on a Functions new UI full screen mode. I'm wondering, if we have a new icon for full screen. If we don't, can I use the old console full screen icon which is sunsetting?

**Comment by richbachman** [comment, marked as answer]
Hi @sang-yeo I'll create a ticket in our backlog to get the full screen icon added. In the meantime, you can use the console one.

Also, not sure if you've ran across it yet but we recently launched a searchable icons page to the docs site: https://paste.twilio.design/icons. This lists all of our current icons, and each have a copy/paste code snippet for engineers.

**Reply by sang-yeo** [reply]
Hey, @richbachman The searchable icons page is cool! 
We will use the console one for now and swap it when it's available. Thanks!

**Reply by richbachman** [reply]
Hi @sang-yeo. Would you mind picking out a Streamline icon to use? Here's more information about that process. https://paste.twilio.design/icons/how-to-add-an-icon/

Thanks.

**Reply by sang-yeo** [reply]
Sure, I picked [icons](https://drive.google.com/file/d/11QxAgTPR7zZcfUlwq6Z0Ig4-lPrqC-lg/view?usp=sharing). Icon names are Expand 1 and Shrink 1. Do you want my developer to use this icon instead?

**Reply by serifluous** [reply]
We'll add them in to Paste, and let you know when it's added. Keep using the console one for now.

**Reply by sang-yeo** [reply]
Will do. Thanks!

**Comment by corinne-nagel** [comment]
Hi @sang-yeo expand and shrink icons are now added and can be viewed here:
https://paste.twilio.design/icons
As a reminder, if you need something on a defined timeline, we would love for you to contribute using the guidelines here: https://paste.twilio.design/icons/how-to-add-an-icon/ Let us know if you have any questions!

**Reply by sang-yeo** [reply]
This is great! We are swapping the old one to the new one. Thank you for a quick turnaround!

**Reply by corinne-nagel** [reply]
Awesome! Glad to hear it 🙌
